tests/test_data/test_data.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+
+
+class MockResponse:
+    def __init__(self, json_data, status_code):
+        self.json_data = json.loads(json_data)
+        self.status_code = status_code
+        self.ok = status_code < 300
+
+    def json(self):
+        return self.json_data
+
+
+dirname = os.path.dirname(__file__)
+
+with open(f'{dirname}/peak_faction_holiday_response.json', mode='r') as data:
+    holiday_response = data.read()
+
+with open(f'{dirname}/peak_faction_workingday_response.json', mode='r') as data:
+    workingday_response = data.read()
```

Comment:
These should only be loaded when they are needed in the test

Reply:
fixed